packages/react-urql/src/hooks/useQuery.ts

```diff
@@ -106,12 +106,38 @@ export function useQuery<Data = any, Variables = object>(
           });
 
           cache.set(request.key, promise);
-          throw promise;
+          const res = {};
```

Comment:
I'm wondering whether we need to match the `OperationResult` more closely here. The type of it can probably help keep this consistent, but I'd guess so 🤔

Reply:
This gets merged with the current operationResult normally, no?

Reply:
Hm, if it was, wouldn't the getter's return value be copied rather than the getter being applied?